myPortfolio/index.html

```diff
@@ -1,25 +1,85 @@
 <!DOCTYPE html>
+
 <html lang="en">
-<head>
+
+  <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-
-  <div class="photoAndIntro">
-
-    <div class="containers">
-        <img id="photoOfMe" src="https://lh3.googleusercontent.com/pw/AIL4fc8XPOMxd2-H5P1N4PIkrsFmAGOcRvGnqW3hqZHg-BHrUM82mIpzK6C1UcZhqgc8GOlGT_hEfViKJqygjI5y7e_Hx_6d6FCAmSsRfTpJcIDxuehLfFxtVKeMGAqlJ3Q__7YfnoLkyVeVzNHhIiV8ky0qrJs9O2wmgojaSvJ0wfB0phK_4BJx6TNQHWM63WC_q4BpKlRvSC4MXG8oJISZbF6UyNLrFx_LY_8n3ZH8FYxBMVLYJDuHm6dH7Y600DXiPafXl-pZxEH6Awl3Jyh3iDKjed1lTYSzxBb9uGF2hf1YOksvXPv2j92TspR5PypZNWU0UKO4lkg-y61r4GpH9ivlgjlEEnrD6bHnBNuh8c_OVy4hqBkPCq0LO2TVsK2VTefK8fxSGX0JFQjN3paUlWvUVewghhDdkqH58zA3a39mWASI4OZecXiqWNMR5rtbz_ppsxVejI6O9BWhyfH_bLM359pXjB7-kPvBrMw5HxA7evf2_VohvzmSGMYMDp0hs4bTMI3Trax0SniZqnksXq9QMviK8B3JgIuF1kUFS5LSPnG0IVyKxnMYPqPelKBhXTbGCufqUSnetdKmrO6nOrJGg81eb8oRw2IQzkepBp4cL84n4-Sgl03P7MfWd3vZOnb6e0uYWgPO2G6jLyEdyx863Pb8MoERzEY8HqOGEre4E9LQ5YgaoNR60m9eA7o6p7WM-RAYM7OMFNkRWdByR4RL8RrtKQ4bXBF-TYbDmv01CvK_WPoboIy8BvRTf04Nf2vEqee28m2YaAaxAXlf0-heA6SpOoWmX8l3vQJG-V9yAOwMEcPR0Fnbs000cWgeALSoXJvTiSAcL4qZH3CMzEE9lu_606ID2YlZlfbVaLFypGQiFKIJnLxPz9x6S9rW3yYURzvDSG78Gi27SRffNA=w1324-h1346-s-no?authuser=0" alt="Photo of Whitney-Rene">
-    </div> 
-
-    <div class="containers">
-        <h1>Whitney-Rene</h1>
-        <h2>Aspiring Software Engineer</h2>
-        <p>Welcome to my portfolio page. ...</p>
-    </div>
+    <title>Whitney-Rene's Portfolio Page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <div class="photoAndBio">
+
+      <div class="containers">
+          <img id="photoOfMe" src="./Whitney-Rene-Chambers-2023-h2.png" alt="Photo of Whitney-Rene">
+      </div> 
+
+      <div class="containers" id="Bio">
+        <div class="headings">
+          <h1>Whitney-Rene</h1>
+          <h2>Software Engineer</h2>
+        </div>
+          <p id="paraFont">Thank you for visiting my portfolio page.  Again, my name is Whitney-Rene, and I am an aspiring software engineer.  I am currently a part of a 6 month with an organization called Techtonica.  Techtonica's mission is to bridge the gap between women of color and the tech industry.  I could not be happier to be a participant in the 2023 H2 Co-Hort. In my free time, I enjoy being outside, connecting with friends and painting my nails!  Please feel free to browse around my webpage.  My contact is listed below.  And please fill out the form below.  I would love to connect! </p>
```

Comment:
`cohort`  
As a non-native speaker, I always need help with grammar, correct usage of words, etc., and suggest everyone use 'Grammarly'. You can add to Chrome as an extension, and the free version is good enough for a software dev need. It'll help a lot when you're writing emails too, and will give you confidence.